src/main/java/org/kitteh/irc/client/library/event/abstractbase/ActorChannelMessageEventBase.java

```diff
@@ -53,13 +51,13 @@
      * Constructs the event.
      *
      * @param client the client
-     * @param originalMessages original messages
+     * @param originalMessage original message
      * @param actor the actor
      * @param channel the channel
      * @param message the message
      */
-    protected ActorChannelMessageEventBase(@NonNull Client client, @NonNull List<ServerMessage> originalMessages, @NonNull A actor, @NonNull Channel channel, @NonNull String message) {
-        super(client, originalMessages, actor, channel);
```

Comment:
maybe document more clearly the diff between getMessage() and getOriginalMessages()

Reply:
https://github.com/KittehOrg/KittehIRCClientLib/issues/263